Lib/glyphsLib/builder/groups.py

```diff
@@ -140,12 +140,28 @@ def _to_glyphs_kerning_group(self, name, glyphs):
 
 def _glyph_kerning_attr(glyph, side):
     """Return leftKerningGroup or rightKerningGroup depending on the UFO
-    group's side (1 or 2).
+    group's side (1 or 2) and writing script.
     """
-    if int(side) == 1:
-        return "rightKerningGroup"
-    else:
-        return "leftKerningGroup"
+    if glyph.parent.format_version > 2:  # Glyphs 3
+        glyph_data = glyphdata.get_glyph(glyph.name)
+        if (
+            glyph_data.script in ("arabic", "hebrew")
+            and glyph_data.category != "Number"
```

Comment:
btw, there other characters belonging to RTL scripts (e.g. Arabic) that are _not_ "Numbers" (in the unicode category sense of Nd, No, etc.) but which have a (weak) left-to-right bidirectional type: e.g. arabic decimal/thousand separators (Po: Punctuation other), or arabic number sign (Cf: Format), and some others.

In ufo2ft we don't check the category is Number, but we check if the bidiType (`unicodedata.bidirectional`) is either "L" (strong left to right), "AN" (Arabic numbers) or "EN" (extended arabic-indic digits)

Reply:
Note (to self, nostly) that in ufo2ft, the kerning pairs for the (LTR) arabic numbers are added to the same `kern_rtl` lookup where the rest of the (RTL) kern pairs between arabic non-numbers are; but they are treated as left-to-right, hence only the x-advance is modified (not also the x-origin):

E.g. see this test case:
https://github.com/googlefonts/ufo2ft/blob/ad28eea062e0dd48678309bd9ef86dfcc85fa85a/tests/featureWriters/kernFeatureWriter_test.py#L691-L695

Whereas here, it looks like the `kerningRTL` dict only contains the RTL kerning pairs.

So ufo2ft's `kern_rtl` lookup and Glyphs3 `kerningRTL` don't exactly overlap; the former is a superset of the latter.